components/FilterCategory/FilterCategory.js

```diff
@@ -19,5 +31,21 @@ export default function FilterCategory({ filterNames, category }) {
 }
 
 const StyledLabel = styled.label`
-  padding: 2px;
+  border-radius: 8px;
+  padding: 4px 4px 4px 8px;
+  margin: 6px 20px 0px 0px;
```

Comment:
Removed unnecessary units
```suggestion
  margin: 6px 20px 0 0;
```